PizzaBuilder/PizzaBuilder/Program.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using System.Collections.Generic;
+namespace PizzaBuilder
+{
+    public interface IBuilder
+    {
+        void BuildDough();
+        void BuildSauce();
+        void BuildTopping();
+    }
+    public class WhitePizzaBuilder : IBuilder
+    {
+        private WhitePizza _whitepizza = new WhitePizza();
+        public WhitePizzaBuilder()
+        {
+            this.Reset();
+        }
+        public void Reset()
+        {
+            this._whitepizza = new WhitePizza();
+        }
+        public void BuildDough()
+        {
+            this._whitepizza.Add("Prepare the dough");
+        }
+        public void BuildSauce()

```

Comment:
It would be nice if you specified some sauces like (White sauce, red sauce) or what you want rather than just writing Add the sauce.